.github/ISSUE_TEMPLATE/installation_issue.md

```diff
@@ -0,0 +1,27 @@
+---
+
+name: Installation Issue 
+about: Ask about installing geopandas 
+title: ""
```

Comment:
Do we want a suggested title here with `INST` or the like?

Reply:
I think that label will be enough. These will be mostly temporary anyway.